data/mods/Magiclysm/techniques.json

```
@@ -14,5 +14,33 @@
     ],
     "description": "Cut damage multiply by 2, crit only",
     "attack_vectors": [ "vector_null" ]
+  },
+  {
+    "type": "technique",
+    "id": "VIKTORHAND_MANATOUCHED_BLAST",
+    "name": "magichand manablast",
+    "//": "supposed to simulate a chaser attack. You attack, and then your mana-hand pitches in another blast.",
+    "melee_allowed": true,
+    "unarmed_allowed": true,
+    "reach_ok": true,
+    "crit_ok": true,
+    "weighting": 5,
+    "eocs": [
+      {
+        "id": "tec_VITKORHAND_MANATOUCHED_BLAST",
+        "effect": [
+          { "math": [ "damagetracker1 = u_val('mana')/500" ] },
+          { "math": [ "damagetracker2 = u_skill('spellcraft')" ] },
+          { "math": [ "_damagetracker3 = damagetracker1 + damagetracker2" ] },
+          {
+            "u_message": "Your appendage blasts the <npc_name> for <context_val:damagetracker3> damage!",
+            "type": "good"
+          },
```

you made damagetracker1 and damagetracker2 global vars, but made damagetracker3 a context var instead
not saying that if you decided to unify and use only pure damage, there is little reason to split variable in two
```suggestion
        "id": "tec_VITKORHAND_MANATOUCHED_BLAST",
        "effect": [
          { "math": [ "_damagetracker = u_val('mana')/500 + u_skill('spellcraft')" ] },
          {
            "u_message": "Your appendage blasts the <npc_name> for <context_val:damagetracker> damage!",
            "type": "good"
          },
```

---

> you made damagetracker1 and damagetracker2 global vars, but made damagetracker3 a context var instead not saying that if you decided to unify and use only pure damage, there is little reason to split variable in two

I got an error about EOFs & string delimiters if I tried something like "damagetracker = (u_val('mana')/500)+u_skill('spellcraft')" so I just split it up & remerged it later.

Trying your edits, it works great. I don't have the error saved, though, so I can't see what exactly I did wrong again.